ts/session/apis/snode_api/swarm_polling_config/SwarmPollingUserConfig.ts

```diff
@@ -0,0 +1,33 @@
+import { ConfigMessageHandler } from '../../../../receiver/configMessage';
+import { RetrieveMessageItemWithNamespace } from '../types';
+
+async function handleUserSharedConfigMessages(
+  userConfigMessagesMerged: Array<RetrieveMessageItemWithNamespace>
+) {
+  try {
+    if (userConfigMessagesMerged.length) {
+      window.log.info(
+        `received userConfigMessagesMerged count: ${userConfigMessagesMerged.length}`
```

Comment:
Do we want these logged everytime publically? Not hidden behind .debug or something? No issue with what is logged just thinking of it clogging up the log file

Reply:
Yeah, I think we should keep those as info